src/main/webapp/components/query-manager/add-query.tsx

```diff
@@ -54,10 +52,9 @@ export default (props: AddQueryProps) => {
         query={query}
         QueryEditor={QueryEditor}
         onSearch={() => {
-          onChange(query)
-          onSearch(query.id!)
+          onCreate(query)
           handleClose()
-          setQuery({ id: new Date().toString() })
+          setQuery({})
```

Comment:
:question: What is this for?

Reply:
We want the add query popover to reset its query state when the query is added